### PR TITLE
Adding Support for Node 10

### PR DESCRIPTION
**Pull request by ncheikh**
#### What?

Updating stencil styles dependency which allows node 10 support.

#### Tested
- [x] Locally against cornerstone & node 8
- [x] Locally against cornerstone & node 10

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "CLI tool to run BigCommerce Stores locally for theme development.",
   "main": "index.js",
   "engines": {
-    "node": ">= 8.0.0 <12.0.0"
+    "node": ">= 8.0.0 <11.0.0"
```
Comment (junedkazi) [inline package.json]: This should be `<12` to support `11.x`

**Comment by xzyfer** [comment]
>Waiting on: dlmanning/gulp-sass#748 for node 12

No need to wait. Node Sass already supports node 12.

File: .travis.yml [hunk 2]
```diff
@@ -7,6 +7,8 @@ addons:
     - github.com 
 node_js:
   - 8
+  - 10
```
Comment (junedkazi) [inline .travis.yml]: Should we add 9 as well ?

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "CLI tool to run BigCommerce Stores locally for theme development.",
   "main": "index.js",
   "engines": {
-    "node": ">= 8.0.0 <9.0.0"
+    "node": ">= 8.0.0 <11.0.0"
```
Comment (junedkazi) [inline package.json]: Also can we also update the readme to reflect node version support.

Comment (ncheikh) [reply package.json]: Good catch!

File: .travis.yml (hunk 2 above)
Comment (ncheikh) [reply .travis.yml]: I don't think so, odd versions are only supported for a short time and I don't think we need to support node versions that aren't supported for cli.  In the future I can see doing the leading version like 13 for early adopters and it helps us make sure we are in a good position for the next even/lts release.

Thoughts?

@junedkazi @bookernath @bc-williamkwon 

https://nodejs.org/en/about/releases/

File: README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 The BigCommerce server emulator for local theme development.
 
 ## Install
-_Note: Stencil requires the Node.js runtime environment, version 8.x. We do not yet have support for Node 10 or greater._
+_Note: Stencil requires the Node.js runtime environment, version 8.x or 10.x We do not yet have support for Node 12 or greater._
```
Comment (ncheikh) [inline README.md]: @junedkazi @bookernath Let me know if we want different messaging here.

**Pull request merged**